### PR TITLE
feat: update archetype to not use facets config

### PR DESCRIPTION
**Pull request by ajperezbau**
Also use facets model name in the custom facets component slots.

[EX-7259](https://searchbroker.atlassian.net/browse/EX-7259)

### CODE REVIEW

File: src/components/search/facets/custom-facets.vue [hunk 1]
```diff
@@ -32,7 +32,7 @@
       </CustomHeaderTogglePanel>
     </template>
 
-    <template #category-paths="{ facet, selectedFilters }">
+    <template #hierarchical-facet="{ facet, selectedFilters }">
```
Comment (tiborux) [inline src/components/search/facets/custom-facets.vue:35]: This is not right, the slot uses the id of the facet:

image

So you have to rollback these names

**Comment by tiborux** [comment]
Update `x-platform-adapter` version

**Review by annacv — COMMENTED** [review]
Hey, @tiborux I'm not sure about this, if we used the `modelName`, wouldn't we avoid repeating the template for facets that share that model??? Maybe I'm missing something... are we constrained to use the `id` for some reason?

**Comment by tiborux** [comment]
> Hey, @tiborux I'm not sure about this, if we used the `modelName`, wouldn't we avoid repeating the template for facets that share that model??? Maybe I'm missing something... are we constrained to use the `id` for some reason?

Yes, you are right, I didn't check the component deeply, I saw that we use the id for the slots, but we also use the model name for the slots,

**Review by alvarodE — APPROVED** [review]

**Review by alvarodE — APPROVED** [review]

**Review by tiborux — APPROVED** [review]

**Pull request merged**